config/api.yml

```diff
@@ -2237,12 +2237,32 @@
     :klass: ChargebackRateDetailMeasure
   :metric_rollups:
     :description: Metric Rollups
-    :identifier: metrics
+    :identifier: metric_rollups
```

Comment:
Are we suggestion there will be 2 separate features for RBAC?  I'm wondering if it's just simple to have one feature that applies to both metrics and rollups.

Reply:
`less is more` ™️